src/twisted/newsfragments/11793.removal

```diff
@@ -0,0 +1 @@
+Twisted Conch has deprecated but still supports legacy ciphers (CAST5, SEED, IDEA, Blowfish) as the upstream library "cryptography" has deprecated them since version 37.
```

Comment:
Are we deprecating SEED and IDEA? From the code I see only CAST5 and Blowfish.

Maybe the release notes can be:

twisted.conch.ssh.transport has the CAST5 and Blowfish based cipher deprecated. This was done since the same ciphers were deprecated in the `cryptography` library since version 37.

Reply:
> Are we deprecating SEED and IDEA? From the code I see only CAST5 and Blowfish.

`cryptography` deprecates all 4 of these ciphers but I only found use of `blowfish` and `cast5` in twisted source code.

I've updated with your notes.